TLM/TLM/Manager/Impl/LaneConnection/LaneConnectionSubManager.cs

```diff
@@ -462,7 +500,9 @@ public class LaneConnectionSubManager :
                 return;
             }
 
-            if (!HasOutgoingConnections(laneId, startNode)) {
+            const bool RESET = false; // reset lane arrows when last lane connection is removed.
+
+            if (RESET && !HasOutgoingConnections(laneId, startNode)) {
```

Comment:
This `if (false &&` is probably a source of a compiler warning

Reply:
it does not (and should not) cause compiler warnings. its a legit thing to do.
we do that all the time when logging:
```cs
#if DEBUG
        private bool verbose_ => DebugSwitch.LaneConnections.Get();
#else
        private const bool verbose_ = false;
#endif
if(verbose_ ) ....
```

Reply:
but I understand if you feel like its unnecessary code